packages/cosmic-swingset/test/inquisitor.test.ts

```diff
@@ -82,3 +109,129 @@ test('smoke test', async t => {
     );
   }
 });
+
+// This is really a test of makeCosmicSwingsetTestKit, but inquisitor relies
+// upon that so we include it here.
+test('vat lifecycle', async t => {
```

Comment:
for reference, `Error: replay should not see "shutdown-worker"` was the symptom from when we tried to use `psm-core-eval.js` that has an `E(adminNode).upgradeContract(...)` call in it.

I'd feel better if I understood how this test relates; would it fail with those symptoms before the fix in this PR and pass after?

Reply:
Yes, either that or a complaint about a missing bundle (depending upon what preceded attempted use of the core eval).